airflow/api_fastapi/core_api/routes/public/task_instances.py

```diff
@@ -339,6 +339,8 @@ def get_task_instances(
     responses=create_openapi_http_exception_doc([status.HTTP_404_NOT_FOUND]),
 )
 def get_task_instances_batch(
+    dag_id: Literal["~"],
+    dag_run_id: Literal["~"],
```

Comment:
I would have assumed it could have acted as a "filter" i.e. get me all TIs for a specific DAG or Run_id, do we not want to allow that?

Reply:
We already have that. That's [Link](https://airflow.apache.org/docs/apache-airflow/stable/stable-rest-api-ref.html#operation/get_task_instances)

This is a batch endpoint to get all TI's of all dags